spec/jobs/facilities/state_cemetery_reload_job_spec.rb

```diff
@@ -65,4 +65,10 @@
     expect(cemetery.phone['main']).to eq('251-625-1338')
     expect(cemetery.phone['fax']).to eq('251-626-9204')
   end
+
+  it 'should gracefully handle missing mailing addresses' do
```

Comment:
I expected a test for this change to test that an empty hash was returned.  I think tests are more valuable when they are more explicit, because it'll help think through future changes that break it.  Is there a reason it's so broadly defined?

Reply:
We went with the not nil check because the value being nil was what caused the error we were fixing and we thought being explicit about the result we wanted to avoid was useful. We discussed checking that an empty hash was the returned instead of/in addition to not nil, so we're not opposed to modifying the test.

Reply:
Explicitly checking for the empty hash makes the most sense. Will modify!